docs/getting-started/configuration.md

```diff
@@ -187,10 +189,77 @@ components: {
  },
 ```
 
+### Add sign-in resolver in the backend
+
+The Auth backend plugin is responsible for assigning a [Backstage User Identity](../auth/identity-resolver.md#backstage-user-identity) when a user signs in. The identity is mainly used in determining the ownership of Backstage Catalog entities for the user. This is achieved by writing a [Sign In Resolver](../auth/identity-resolver.md#sign-in-resolvers) which takes care of assigning the right identity to the user and even reject SignIn attempts from unrecognized email domains.
```

Comment:
The second sentence could be adjusted a bit. The identity part of it (the `sub`) is your identity and that identifies your user for any purpose within the Backstage ecosystem. It can for example be used as a primary key in a per-user settings backend, or any other similar purpose. The `ent` is what relates to ownership (the list of entity refs through which you claim ownership).

Reply:
Hmm, I was afraid of being too technical in this line and give a one-liner for them to read and move on with auth ready. The Sign In Resolver page talks about `sub` and `ent` in good detail.

But let me adjust here per your suggestion!